directories/src/redux/epics/node-epics.js

```diff
@@ -153,6 +161,47 @@ const requestGenesisHashEpic = (action$, store) => {
     });
 };
 
+const treasureHuntEpic = (action$, store) => {
+  return action$
+    .ofType(nodeActions.TREASURE_HUNT)
+    .mergeMap((action) => {
+      const { genesisHash, numberOfChunks } = action.payload;
+      const datamap = Datamap.generate(genesisHash, numberOfChunks);
```

Comment:
I think this won't work.  The DataMap.generate method expects a handle, not a genesis hash.  I believe in a future PR @jfarago will be updating the npm package for the datamap to accept a genesis hash instead of a handle.

Reply:
I think there's a method in the encryption file that does it.  Think the method is called "genesisHash"